dhis-2/dhis-api/src/main/java/org/hisp/dhis/security/utils/CspConstans.java

```diff
@@ -30,9 +30,9 @@
 import java.util.List;
 import java.util.regex.Pattern;
 
-public class CspUtils
+public class CspConstans
```

Comment:
Typo - references will also need to be updated.

```suggestion
public class CspConstants
```